### PR TITLE
[tests-only] add link share info getter

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description

Add a link share information getter from the provided attribute name.

## Related Issue

- Part of https://github.com/owncloud/activity/issues/800

## How Has This Been Tested?

- :robot:

### CODE REVIEW

File: tests/acceptance/features/bootstrap/Sharing.php [hunk 1]
```diff
@@ -3759,6 +3759,37 @@ public function getLastPublicShareToken():string {
...
 
+	/**
+	 * Returns the attribute values from the last public link share data
+	 *
+	 * @param $attr - attribute name to get
+	 *
+	 * @return string
+	 * @throws Exception
+	 */
+	public function getLastPublicShareAttribute($attr): string {
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/Sharing.php:3770]:
```suggestion
	public function getLastPublicShareAttribute(string $attr): string {
```

These days we declare types everywhere that we can.

**Comment by SwikritiT** [comment]
The CI is green for this PR so this https://github.com/owncloud/core/issues/40436 failure is intermittent?

**Comment by SwikritiT** [comment]
> The CI is green for this PR so this #40436 failure is intermittent?

nevermind is because of the lack of full-ci

**Comment by phil-davis** [comment]
> The CI is green for this PR so this #40436 failure is intermittent?

https://drone.owncloud.com/owncloud/core/37073/27/5

"skip-if-unchaged" - the PR title did not say "full-ci" and it only changed acceptance test code, so PHP unit tests are skipped.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**